storage/app/tests/unit/test_storage.py

```diff
@@ -0,0 +1,19 @@
+import os
+import unittest
+import warnings
+from app.tests.ordered_test_helper import ordered
+
+
+class TestStorage(unittest.TestCase):
+    test_file_path = f'../test_files/cif-test-document.docx'
+
+    def setUp(self) -> None:
+        warnings.simplefilter("ignore")
+
+    @ordered
+    def test_upload_file(self):
+        pass
```

Comment:
I guess we can add those later, but they'll need to be written for our user stories (next sprint) to pass the DoD